Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -10,11 +9,12 @@ namespace Kudu.Contracts.SiteExtensions
     // This is equivalent to NuGet.IPackage
     public class SiteExtensionInfo : INamedObject
     {
-        public enum SiteExtensionType
+        public static class SiteExtensionType
```

Comment:
Can't it still be an Enum and then use `StringEnumConverter`? See [here](http://stackoverflow.com/questions/2441290/json-serialization-of-enum-as-string). The advantage is that it avoids having to deal with strings throughout the code.